src/server/game/Globals/AreaTriggerDataStore.h

```diff
@@ -19,15 +19,20 @@
 #define AreaTriggerDataStore_h__
 
 #include "Define.h"
+#include "ObjectMgr.h"
 
 class AreaTriggerTemplate;
 class AreaTriggerMiscTemplate;
+struct AreaTriggerServerPosition;
 
 class TC_GAME_API AreaTriggerDataStore
 {
 public:
     void LoadAreaTriggerTemplates();
 
+    CellGuidSet const* GetAreaTriggersForMapAndCell(uint32 mapId, uint32 cellId) const;
```

Comment:
If you are not storing this in CellObjectGuids then just use std::set<ObjectGuid::LowType> directly, not with typedef (tl;dr don't include ObjectMgr.h in headers)